docs/guide-uk/start-hello.md

```diff
@@ -22,7 +22,7 @@
 параметр `message` із запиту і відображає його значення користувачу. Якщо в запиті
 відсутній параметр `message`, то дія буде відображати "Привіт".
 
-> Info: [Дії](structure-controllers.md#creating-actions) являються об’єктами, на які користувач може посилатись для
+> Інформація: [Дії](structure-controllers.md#creating-actions) являються об’єктами, на які користувач може посилатись для
```

Comment:
No, that should not be changed. See https://github.com/yiisoft/yii2/blob/master/docs/documentation_style_guide.md#blocks

Reply:
Did I understand correctly, can I change it to "info" everywhere?

Reply:
It needs to be `Info` everywhere with translated versions of it put in `blocktypes.json` file.